gapis/api/gles/compat.go

```diff
@@ -213,17 +212,15 @@ func compat(ctx context.Context, device *device.Instance) (transform.Transformer
 			// TODO(dsrbecky): This might make some atoms valid for replay which were invalid on trace.
 			scs := FindStaticContextState(cmd.Extras())
 			if scs != nil && !version.IsES && scs.Constants.MajorVersion < 3 {
-				clone, err := deep.Clone(cmd)
-				if err != nil {
-					panic(err)
-				}
-				cmd = clone.(*EglMakeCurrent)
-				for _, e := range cmd.extras.All() {
+				clone := cb.EglMakeCurrent(cmd.Display, cmd.Draw, cmd.Read, cmd.Context, cmd.Result)
+				clone.Extras().MustClone(cmd.Extras().All()...)
```

Comment:
I have been bitten by the same deep.Clone "bug" in the past - maybe deep.Clone should fail instead?